<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcedureNavigationWidget</class>
 <widget class="QWidget" name="ProcedureNavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="navigation_widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="steplabel4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第4步</string>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>229</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>157</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>0</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>料号建档流程</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第2步</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>106</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>127</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label7">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>331</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第6步</string>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>413</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPushButton">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>495</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>下一步</string>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>178</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label8">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>382</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousPushButton">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>495</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>上一步</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第9步</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第5步</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第1步</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>259</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>208</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>76</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icon/processing.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>390</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第8步</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>25</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icon/processing.png</pixmap>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>362</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第7步</string>
      </property>
     </widget>
     <widget class="QLabel" name="steplabel3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>第3步</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>55</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label9">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>433</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>311</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label6">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>280</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainUIPushButton">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>495</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>主界面</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
